### PR TITLE
issue: Update README for [#76] Could not get script via 'curl -vsS https://raw.githubusercontent.com/hyperledger/fabric/master/scripts/bootstrap.sh | bash'

### PR DESCRIPTION
**Pull request by lyp830414**
[#76] Could not get script via 'curl -vsS https://raw.githubusercontent.com/hyperledger/fabric/master/scripts/bootstrap.sh | bash'

solution:
	Update README.MD & README_ZH.MD, remarking that manually download the script instead of CURL
Signed-off-by: lyp830414 <lyp830414@163.com>

### CODE REVIEW

File: README.MD [hunk 1]
```diff
@@ -27,15 +27,29 @@ To answer, the best parameter for specific fabric network and fabric chaincode,
 - GUI for tape, tape focus on once off time performance testing.
 
 ## Quick start
-1. install this project `npm install`
-1. install fabric sample `curl -vsS https://raw.githubusercontent.com/hyperledger/fabric/master/scripts/bootstrap.sh | bash`
-1. get tape `docker pull guoger/tape`
-1. apply the bridge file to adjust block parameters for test network `cp sample/prepareConfig.sh fabric-samples/test-network`
+1. Install this project `npm install`
+
+1. Install fabric sample `curl -vsS https://raw.githubusercontent.com/hyperledger/fabric/master/scripts/bootstrap.sh | bash`
```
Comment (SamYuan1990) [inline README.MD:32]: 这里和下边的步骤是或的关系。这里需要调整一下。

1. Visit and download the fabric environment installation script :

File: README_ZH.MD [hunk 2]
```diff
@@ -25,6 +25,8 @@ Probe是在[TWGC performance work group](https://github.com/Hyperledger-TWGC/fab
 ## 快速上手
 1. 安装项目 `npm install`
 1. 下载fabric-sample `curl -vsS https://raw.githubusercontent.com/hyperledger/fabric/master/scripts/bootstrap.sh | bash`
+1. 下载fabric环境安装脚本`https://raw.githubusercontent.com/hyperledger/fabric/master/scripts/bootstrap.sh`
```
Comment (SamYuan1990) [inline README_ZH.MD:28]: 同理

**Review by SamYuan1990 — APPROVED** [review]
建议修改一下，这两个脚本实际上是一个脚本存放的形式不同而已。因此是or逻辑 而不是先后次序的关系。

**Review by SamYuan1990 — CHANGES_REQUESTED** [review]
changes needed

**Comment by lyp830414** [comment]
Now updated README.md & README_ZH.md, please have a review @SamYuan1990

**Comment by lyp830414** [comment]
Updated again and added some note information for README. @SamYuan1990

**Comment by lyp830414** [comment]
Close due to signature & commits are confused

**Pull request closed without merging**